agent-control/src/agent_control/run/k8s.rs

```diff
@@ -207,6 +213,20 @@ impl AgentControlRunner {
             self.k8s_config.current_chart_version.clone(),
             self.k8s_config.cd_release_name,
         );
+        let (agent_control_internal_publisher, agent_control_internal_consumer) = pub_sub();
+
+        let _cd_version_checker = spawn_version_checker(
+            AgentID::K8sCD,
+            HelmReleaseVersionChecker::new(
+                k8s_client,
+                helmrelease_v2_type_meta(),
+                self.k8s_config.namespace.clone(),
+                &AgentID::K8sCD,
```

Comment:
The agent_id for the `HelmReleaseVersionChecker` is actually the name of the HelmRelease. Would it make sense to update the terms in the `HelmReleaseVersionChecker` and avoid introducing the `AgentID::K8sCD`?

As I see it, the agent_id should corresponding to `agent-control-cd` should be `AgentID::AgentControl` and introducing a new one can be misleading.

Reply:
Changed!